### PR TITLE
[BPK-4210] Update Dialog design

### PR DESCRIPTION
**Pull request by k0nserv**
Now uses large corner radius and a small body text.

This also impacts the bottom sheet variant

Simulator Screen Shot - iPhone 8 - 2020-09-28 at 10 53 08

I want to get Dima's input on this when he's back from holiday.

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/master/Example/Backpack%20Screenshot/Screenshots.swift)
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/master/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack/Dialog/Classes/BPKDialogView.m [hunk 1]
```diff
@@ -111,7 +111,7 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
 - (void)setupViews {
     self.backgroundView = [[UIView alloc] initWithFrame:CGRectZero];
     self.backgroundView.backgroundColor = self.dialogContentViewBackgroundColor;
-    self.backgroundView.layer.cornerRadius = BPKCornerRadiusSm;
+    self.backgroundView.layer.cornerRadius = BPKCornerRadiusLg;
```
Comment (georgegillams) [inline Backpack/Dialog/Classes/BPKDialogView.m:114]: AFAIK we already have a property for determining the cornerRadius. So how does this affect that?

**Comment by k0nserv** [comment]
This is blocked by https://github.com/Skyscanner/backpack-ios/pull/750 to update screenshots

File: Backpack/Dialog/Classes/BPKDialogView.m (hunk 1 above)
Comment (k0nserv) [reply Backpack/Dialog/Classes/BPKDialogView.m:114]: That is a good question @georgegillams and it probably means this needs some more work and a further discussion with Dima.

**Comment by k0nserv** [comment]
As discussed with Dima offline we'll keep the smal radius for the default. Consumers can still use the large style via `cornerStyle`

**Review by georgegillams — APPROVED** [review]

**Pull request merged**